Dev/nigelhew

nhew1994 opened the pull request:
Update dfttk/eos_fit.py
- Modified def plot_ev and plot_energy_difference
- Also added some TODO comments for future work
![newplot](https://github.com/lukeamyers/vasp-job-automation/assets/77617652/a6508e53-6bdc-4237-b4ca-163cabcedaaa)
![newplot1](https://github.com/lukeamyers/vasp-job-automation/assets/77617652/8e4e0c9a-8a7a-4f8b-b858-28543e5fa8c6)

nhew1994 committed:
Update dfttk/eos_fit.py
- Modified def plot_ev and plot_energy_difference
- Also added some TODO comments for future work

nhew1994 committed:
Typo fixed in eos_fit.py

lukeamyers reviewed (changes requested):
Love it. Looks great!

I would just make the following change.

For handling the per_atom condition, instead of 
```
print("highlight_minimum must be True or False")
sys.exit(1)
```

should probably do:
`raise ValueError("per_atom must be True or False")`

That way errors can be handled as desired.

nhew1994 committed:
Update dfttk/eos_fit.py
In def plot_ev:
Replaced:
print("highlight_minimum must be True or False")
sys.exit(1)

and

print("per_atom must be True or False")
sys.exit(1)

With:
raise ValueError("highlight_minimum must be True or False")

and

raise ValueError("per_atom must be True or False")

lukeamyers reviewed (commented):
perfect thanks!
